layouts/partials/head.html

```diff
@@ -61,7 +61,5 @@
   {{ $enableGtagForUniversalAnalytics := not .Site.Params.disableGtagForUniversalAnalytics -}}
   {{ if (or $enableGtagForUniversalAnalytics (hasPrefix .Site.GoogleAnalytics "G-")) -}}
     {{ template "_internal/google_analytics_gtag.html" . -}}
-  {{ else -}}
-    {{ template "_internal/google_analytics_async.html" . -}}
```

Comment:
Not sure if we actually need this file. @thesuperzapper Do you know?

Reply:
Yeah I mentioned that I was uncertain about this as the fix in my issue.